cli/src/funTest/kotlin/ExamplesTest.kt

```diff
@@ -107,7 +107,7 @@ class ExamplesTest : StringSpec() {
                 model = model,
                 copyrightGarbage = CopyrightGarbage(),
                 licenseConfiguration = LicenseConfiguration(),
-                packageConfigurationProvider = SimplePackageConfigurationProvider()
+                packageConfigurationProvider = SimplePackageConfigurationProvider.EMPTY
```

Comment:
The `SimplePackageConfigurationProvider` is designed in such a way that it can be constructed with an empty set of configurations. So this code change introduces a second way to achieve the exact same thing.

For the other code (where this one is aligned with) it's not possible to create an empty provider by constructing any of the existing classes which is why the `ÈMPTY` value there is assigned to an anonymous implementation - which is basically which is basically a short-form of providing a separate non-anonymous class.

Given the above I'd be happier with this code - but I won't block this change.

Reply:
> The `SimplePackageConfigurationProvider` is designed in such a way that it can be constructed with an empty set of configurations. So this code change introduces a second way to achieve the exact same thing.

If you're referring to the default `emptyList()` argument in the constructor, my change is also removing that.

Reply:
No, I'm not.

Reply:
What confused me with the original code was that `SimplePackageConfigurationProvider()` was used by default, which makes it sound (from the name of the class) as if it actually *does* something (something simple, but meaningful), whereas the truth is that nothing is done. From that perspective I find `SimplePackageConfigurationProvider.EMPTY` much clearer.

Reply:
> No, I'm not. I'm referring to the fact that it can be constructed with an empty set of curations.

But that's true for many other classes that already have `EMPTY`, like e.g. `Identifier`: Of course you can manually construct an empty instance by explicitly passing empty values. `EMPTY` is just convenience for that. So, I'm sorry, but I truly do not get your point.

Reply:
Well, the commit message was reasoning that it would align it with other providers. So I compared it with other providers like e.g. `PackageCurationProvider` but not with `Identifier`. To me comparing it with `Identifier` changes the scope of the discussion. 

Anyhow, let's just disagree at this point. It was and still is just a slight preference on my side, but I don't have any issues with following your preference.

Reply:
> Well, the commit message was reasoning that it would align it with other providers.

It didn't, as it said "To align with similar code / classes that already have EMPTY."

Reply:
I believe "similarity of code / classes" just leaves room for interpretation, so it's not that clear IMO.

Reply:
Correct, I deliberately was not specific / referring to provider classes only, but to all classes that already have `EMPTY`.